src/main/java/com/example/onlinebookstore/model/Book.java

```diff
@@ -0,0 +1,30 @@
+package com.example.onlinebookstore.model;
+
+import jakarta.annotation.Nonnull;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Nonnull
```

Comment:
@Nonnull annotation is used for validation purposed, not as a constraints of database.

If you want to add nullable constraints you need to use @Column(nullable=false)